### ISSUE TITLE
x/tools/cmd/goimports: No longer compiles on go1.4

### ISSUE THREAD

**Issue by freeformz** [golang/go#16661]
Please answer these questions before submitting your issue. Thanks!

1. What version of Go are you using (`go version`)?
go1.4

2. What operating system and processor architecture are you using (`go env`)?
Linux

3. What did you do?

```console
$ go get golang.org/x/tools/cmd/goimports
package golang.org/x/tools/cmd/goimports
	imports runtime/trace: unrecognized import path "runtime/trace"
```

4. What did you expect to see?

```console
$ $GOPATH/bin/goimports
```

5. What did you see instead?

```console
   imports runtime/trace: unrecognized import path "runtime/trace"
```

Seen here: https://travis-ci.org/lib/pq/jobs/151114204

This is because of https://github.com/golang/tools/commit/edf8e6fef861d8c5ecc9e394b0146d6ebba57795

go1.4 isn't *actually* supported anymore, but people do still use it, that's why `lib/pq` tests against it.

If there is interest in "fixing" this I can submit a CL, otherwise `lib/pq` will either have to remove the goimports test and/or stop testing against go1.4

**Comment by cespare** [comment]
Dup of #16434

**Comment by bradfitz** [comment]
Dup of #16434

We've decided to not fix it, since we don't test Go 1.4 ourselves at https://build.golang.org/ anymore.

**Comment by freeformz** [comment]
Sorry, I only searched open issues. :-(
